backend/models.py

```diff
@@ -75,22 +100,29 @@ class Meta:
         )
 
 
-class ParagraphsTranslateRequest(BaseModel):
-    paragraphs: list[str]
-    source_language: str
-    target_language: str
-
-
 class Provider(str, Enum):
     DEFAULT_DEV = "dev"
     SIMPLE_GPT_1 = "simple-gpt-1"
     OPENAI = "openai"
 
+class Example(BaseModel):
+    sourceText: str
+    firstTranslation: str
+    lastTranslation: str
+
+class ParagraphsTranslateRequest(BaseModel):
+    paragraphs: List[str]
+    source_language: str
+    target_language: str
+    dictionary_id: int | None = None
+    dictionary_timestamp: str | None = None
+    examples: List[Example] | None = None
```

Comment:
We should move examples inside dictionary and remove `examples` from `TranslateRequest`

Reply:
Just to confirm – are you suggesting we stop sending examples from the frontend and fetch them in the backend based on the dictionary instead? Our current flow is to include them in the request